Cabal/src/Distribution/PackageDescription/Check.hs

```diff
@@ -1018,12 +1015,11 @@ checkMissingDocs dgs esgs edgs efgs = do
 
         -- 2. Realise Globs.
         let realGlob t =
-              concatMap globMatches
-                <$> mapM (runDirFileGlobM ops "") t
-        rgs <- realGlob dgs
-        res <- realGlob esgs
-        red <- realGlob edgs
-        ref <- realGlob efgs
+              concatMap globMatches t
```

Comment:
can simplify this to `globMatches t` and rely on callers to `concat` the args, too 🤷🏻